build.rs

```diff
@@ -17,8 +17,13 @@ fn main() {
     let path_maildir: &str =
         get_first_path(&["/var/mail", "/var/spool/mail", "/usr/spool/mail"]).unwrap_or("/var/mail");
 
-    // TODO: use _PATH_STDPATH from paths.h
-    println!("cargo:rustc-env=PATH_DEFAULT=/usr/bin:/bin:/usr/sbin:/sbin");
+    // TODO: use _PATH_STDPATH and _PATH_DEFPATH_ROOT from paths.h
```

Comment:
this breaks some existing sudo tests:
``` console
    env_reset::some_vars_are_set
    env_reset::user_dependent_vars
    env_reset::vars_whose_values_start_with_parentheses_are_removed
```

I think sudo and su have different default PATH values